compiler/control/CompileMethod.cpp

```diff
@@ -379,6 +383,17 @@ compileMethodFromDetails(
          // not ready yet...
          //OMR::MethodMetaDataPOD *metaData = fe.createMethodMetaData(&compiler);
 
+         if (compiler.compileRelocatableCode())
+            { 
+            const char* methodName = compilee.externalName(&trMemory);
+            TR::Compiler->aotAdapter->createAndRegisterAOTMethodHeader(
+                        methodName,
+                        startPC,
+                        compiler.cg()->getCodeLength(),
+                        compiler.cg()->getAheadOfTimeCompile()->getRelocationData(),
+                        compiler.cg()->getAheadOfTimeCompile()->getSizeOfAOTRelocations());
+            }
+
```

Comment:
This is done again below; did you mean to delete this?

Reply:
Hi, thank you for the comment, this shall be fixed. I believe it is a merging artifact.

Reply:
I've fixed it, thank you!